Traceback (most recent call last):
  File "/WAVE/users/unix/sli13/workspace/traffic_assignment/test.py", line 21, in <module>
    runner.compare_xfc_result_vary_proning()
  File "/WAVE/users/unix/sli13/workspace/traffic_assignment/runner.py", line 198, in compare_xfc_result_vary_proning
    json.dump(locs, f)
  File "/WAVE/users/unix/sli13/.conda/envs/traf/lib/python3.12/json/__init__.py", line 179, in dump
    for chunk in iterable:
  File "/WAVE/users/unix/sli13/.conda/envs/traf/lib/python3.12/json/encoder.py", line 432, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)
  File "/WAVE/users/unix/sli13/.conda/envs/traf/lib/python3.12/json/encoder.py", line 406, in _iterencode_dict
    yield from chunks
  File "/WAVE/users/unix/sli13/.conda/envs/traf/lib/python3.12/json/encoder.py", line 326, in _iterencode_list
    yield from chunks
  File "/WAVE/users/unix/sli13/.conda/envs/traf/lib/python3.12/json/encoder.py", line 439, in _iterencode
    o = _default(o)
        ^^^^^^^^^^^
  File "/WAVE/users/unix/sli13/.conda/envs/traf/lib/python3.12/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Node is not JSON serializable
